### DISCUSSION TITLE
Protocol definitions for rs485, display, wifidongle ?

### DISCUSSION THREAD

**Discussion by PepeTheFroggie** [syssi/esphome-soyosource-gtn-virtual-meter#155] [General]
Is there a readable form of the soyosource protocol definitions?
Kinda sux digging that out of C code or assembling it from different forum posts.

I did initally use esp8266 to replace a broken LCD, stopped as soon as i had the soyo working again. Now i got a replacement soyo and may tinker a bit with battery storage. Since RS485 sux i intend to bypass the rs485 chip. 

Is it possible to feed limiter msg over the display/dongle rs232 or does it work only over the rs485 port?

**Comment by syssi** [comment]
You cannot feed the limiter using the display port. Updating the power limit setting periodically isn't a good idea because it will destroy the flash/eeprom some day. Using the RS485 is the recommended way to control the inverter.
